app/templates/skeleton/drupal-org.make

```diff
@@ -97,6 +97,12 @@ projects[views][version] = "3.13"
 projects[views_bulk_operations][subdir] = "contrib"
 projects[views_bulk_operations][version] = "3.3"
 
+; Themes
+projects[bootstrap][subdir] = "contrib"
+projects[bootstrap][version] = "3.x"
+projects[bootstrap][revision] = 6abdabb
```

Comment:
This is probably not enough - please look at the example in https://www.drupal.org/node/1432374

```
projects[draggableviews][type] = module
projects[draggableviews][download][type] = git
projects[draggableviews][download][revision] = 9677bc18b7255e13c33ac3cca48732b855c6817d
projects[draggableviews][download][branch] = 7.x-2.x
```